ArduPlane/ReleaseNotes.txt

```diff
@@ -1,6 +1,17 @@
+Release 7.0.0beta2 11th November 2024
+------------------------------------------------------------------
+This release fixes issues identified during PIQC and also other minor config changes as listed below :
+
+Fleetwide Changes
+ - STALL_PREVENTION is disabled
+ - Max retrials for engine retrials can be defined by setting param - STRT_MX_RTRY
+
+Technical
+ - Default param for serial protocol for 4G
+ - cx_vtol_info.lua script is embedded into CubeOrangePlus firmware
```

Comment:
```suggestion
Fleetwide Changes
 - STALL_PREVENTION is disabled

Technical
 - Default param for serial protocol for 4G
 - Fixed missing features that didn't carry over from Cx5
     - Max retrials for engine
     - cx_vtol_info.lua script was missing for CubeOrangePlus
```

And in the final Cx7 release notes, we'll delete this section about fixing missing stuff that didn't carry over.

Reply:
PR is merged. Could not merge the above changes.